eng/scripts/compatibility/Check-AOT-Compatibility.ps1

```diff
@@ -80,7 +80,7 @@ $actualWarningCount = 0
 
 foreach ($line in $($publishOutput -split "`r`n"))
 {
-    if ($line -like "*analysis warning IL*")
```

Comment:
The warning is in a little different format than the others I've seen but it's fine because this if statement actually only just gets the number of warnings for the print statement. The actual check for the warnings against the expected warnings pattern matches on IL/d below.  I might just merge the two checks together in a separate PR.